### DISCUSSION TITLE
Map git user.name to author on publish

### DISCUSSION THREAD

**Discussion by adamSneller** [verdaccio/verdaccio#3233] [Q&A]
I am using Lerna to publish packages from my NPM7 monorepo to Verdaccio. So far so good! But I notice that unless I explicitly include the "author" in the `package.json` for each package, Verdaccio shows this as `Anonymous`.

It seems like it would make sense to tie this to whichever Git user is publishing, at the time, so you can have an accurate history of who has published which version...

Is there a way to configure this out-of-the box?

**Comment by edclement** [comment]
Hi Adam, I believe Verdaccio follows the same practices the official npm repository does in this regards, which is to rely on the [author](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors) (as you mentioned) in the `package.json`. 

To accomplish what you are proposing, when you publish a package, the `npm` cli would need to be aware of the Git repository/configuration and then modify the `author` field of the `package.json` when publishing, which doesn't sound like a good idea.
